crates/holochain/CHANGELOG.md

```diff
@@ -7,12 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Adds new functionality to the conductor admin API which returns disk storage information. The storage used by apps is broken down into blobs which are being used by one or more app.
```

Comment:
Why are you adding this to the main changelog directly? It'll make it there under the `holochain_conductor_api` heading if it's that crate's changelog.